astropy/modeling/projections.py

```diff
@@ -793,8 +798,8 @@ class Sky2Pix_CylindricalPerspective(Sky2PixProjection, Cylindrical):
         Radius of the cylinder in spherical radii, λ.  Default is 0.
     """
 
-    mu = Parameter(default=1.0)
-    lam = Parameter(default=1.0)
+    mu = Parameter(default=1.0, description="Distance from center of sphere in the direction opposite the projected surface, in spherical radii")
```

Comment:
```suggestion
    mu = Parameter(default=1.0, description="Distance from center of sphere in spherical radii")
```
I think this may safely be shortened a bit, otherwise the docstring itself would need to be broken into two lines.
But perhaps someone familiar with that model could better comment on what is a sufficient description.